demo/src/Components/Tabs/TabPanels.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import T from 'prop-types'
+
+const TabPanels = ({ activeIndex, children }) => {
+  const filteredChildren = React.Children.map(children, (child, index) => {
+    if (index === activeIndex) {
+      return child
+    }
+    return null
+  })
+  return (
+    <div>
+      {filteredChildren}
+    </div>
+  )
+}
+
+TabPanels.propTypes = {
+  children: T.node,
```

Comment:
We should try and figure out what was going wrong with the `isRequired` flags on our proptypes.